sdk/iot/provisioning/readme.md

```diff
@@ -1,4 +1,4 @@
-# Azure IoT Clients for Embedded C: Azure IoT Device Provisioning Client
+# Azure IoT Device Provisioning Client
```

Comment:
We should correct here as well: https://docs.microsoft.com/en-us/azure/iot-dps/about-iot-dps